_posts/2020-07-01-#18991.md

```diff
@@ -7,16 +7,74 @@ authors: [naumenkogs]
 components: ["p2p"]
 host: naumenkogs
 status: upcoming
-commit:
+commit: 52d22a3c
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- Today's PR is the under-review [#18991: Cache responses to GETADDR to prevent
+  topology leaks](https://github.com/bitcoin/bitcoin/pull/18991).  It caches
+  the ADDR responses to GETADDR messages to protect the privacy of the responder.
+
+- There are 3 main objects being transmitted over the p2p network:
+  transactions, blocks and network addresses (like 172.1.2.3). We will focus on
+  the latter. There are 2 relevant messages: GETADDR (a sender asks to provide
+  him with network addresses the receiver knows along with some per-address
```

Comment:
suggestion: s/a sender asks to provide him with network addresses the receiver knows/the sender requests the receiver for the network addresses they know of,/

Reply:
(the ADDR message description could be improved as well)